projects/proftpd/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/proftpd/proftpd"
+main_repo: "https://github.com/proftpd/proftpd"
+language: c
+primary_contact: ""
+auto_ccs:
+  - "Adam@adalogics.com"
+sanitizers:
+  - address
+  - undefined
+  - memory
```

Comment:
Given the number of uninstrumented external libraries that ProFTPD uses, I would not recommend including the memory sanitizer, at least not initially.